kili/mutations/asset/helpers.py

```diff
@@ -146,6 +148,9 @@ def check_file_mime_type(content: str, input_type: str, verbose: bool = True) ->
     Returns true if the mime type of the file corresponds to the allowed mime types of the project
     """
 
+    if input_type == 'FRAME':
+        warnings.warn("FRAME input type is deprecated. Please use VIDEO instead")
```

Comment:
This is called for every file uploaded though the append_many_to_dataset, if the client uploads 200 assets, he doesn't want to have 200 warnings. + all the other warnings that you raise